### ISSUE TITLE
steps to limit liability

### ISSUE THREAD

**Issue by sbma44** [openaddresses/openaddresses-ops#1]
I spoke with Marcia Hofmann today about legal steps we can take to reduce our liability, should a data provider decide we're not in compliance with their terms. She: 

- endorsed the idea of rolling an LLC
- suggested registering with the DMCA Safe Harbour process. 
- said we were probably pretty safe insofar as we're based in the US and she feels address databases aren't eligible for copyright here

Mapbox recently did the DMCA process recently and it was pretty painless. Here's the form:

http://copyright.gov/onlinesp/agent.pdf

All it takes is a small filing fee (we can cover this) and a designated agent to whom takedown requests will be sent. In return we get various protections relating to claims in US courts. @iandees are you comfortable being the potential recipient of these complaints? If not, say the word and I'm sure someone else will step into the breach.

@NelsonMinar you had mentioned you were interested in the LLC process. Do you feel comfortable running that out? If not I can start to move things on the Mapbox end of things.

cc @NelsonMinar @iandees @migurski

**Comment by NelsonMinar** [comment]
Thanks for following up on this! I like the idea of us following DMCA Safe Harbour process.

I don't have any particular expertise with LLCs; all I did was pay a lawyer several thousand dollars to set one up for me in California a couple years back. I can take it on if we need, but if you know someone who knows about map legal issues already it'd probably be better to work with them.

Is there any reason to worry which state the LLC is registered in? I could imagine it being relevant specifically when it comes to IP liability.

**Comment by sbma44** [comment]
I don't know much about this process. Mapbox's in-house counsel will be available to help with this in a couple of months, so if we are content to let things sit until then, we can push on it.

**Comment by migurski** [comment]
Agreed with all the above. DMCA process is not a thing I know, but love the idea of having a process organized.

**Comment by migurski** [comment]
[Paraphrased from Nolo](http://www.nolo.com/legal-encyclopedia/california-form-llc-31670.html):

1. Choose a Name for Your LLC. Under California law, an LLC’s name must end with “Limited Liability Company” or the abbreviations “LLC” or “L.L.C.”
2. File Articles of Organization with the State, [using form LLC-1](http://bpd.cdn.sos.ca.gov/llc/forms/llc-1.pdf) and paying $70 online, by mail, or in person.
3. Appoint a Registered Agent. This is an individual or corporation that agrees to accept legal papers on the LLC’s behalf if it is sued. An LLC may not serve as its own agent for service of process and only one agent can be designated.
4. Prepare an Operating Agreement. Not required in California, but highly advisable.
5. File a Statement of Information With the State, [using form LLC-12](http://bpd.cdn.sos.ca.gov/llc/forms/llc-12.pdf) and paying $20 by mail or in person.

I can do most of this, but I’m unsure about the registered agent. Does that need to be a lawyer?

**Comment by NelsonMinar** [comment]
I have an LLC in California. I paid a lawyer ~$2000 to set it up, which in retrospect was silly but easier than figuring out how to do it myself. I'm pretty sure you can draft the documents yourself using a template.

A California LLC must also file a couple of things a year; a tax return (even for $0) and an updated statement of information. There's also a $800/year fee for maintaining it in California.

**Comment by migurski** [comment]
Sounds like a thing we could use in a funding pitch.

**Comment by dianashk** [comment]
Hey all, just wanted to chime in here. The Mapzen geocoder ❤️  OpenAddresses, as I'm sure other geocoders do... or should... or will one day soon, if they're smart! We would love to see more resources dedicated to this important work.

It would seem like the first step towards allocating resources towards a thing, is positioning said thing to accept the resources, be they financial or in-kind. I think an LLC would make this more straight-forward and transparent. 

It could also be really cool to run a crowdfunding campaign to raise 💰 to support becoming an LLC and /or ongoing work. I'm sure the community would be excited to contribute to the growth of this project as well. Put me down for `$10`! 🎆 🌈 

You will also probably need stickers... unless you already have them and I'm missing out!

**Comment by iandees** [comment]
Sticker requests go to #13.

**Comment by iandees** [comment]
Can we take a step back a tiny bit and talk about what kind of liabilities we're looking to limit? Is it more than DMCA notices? What does an LLC give us beyond making it easier to find a designated legal contact? Does forming an LLC automatically trigger protections?

At one point, we talked about seeking a parent organization. Would that be a different path to limiting liability?

**Comment by migurski** [comment]
A parent organization can help, certainly. When I think of reasons for forming an LLC, I’m thinking mostly of these:

* A liability shield, as we’ve discussed.
* An entity that can accept resources or donations.
* A formal structure that can make strategic decisions.

**Comment by NelsonMinar** [comment]
Note that creating a non-profit is way, way harder than creating an ordinary LLC. We don't have to be a non-profit to take donations, but we do if we want everything to be tax deductible.

OpenAddresses either needs to decide to be a formal thing with decision making processes, etc, or else just stay the casual path we're on now. But either way I think it'd be great if we could shelter inside some larger LLC, in an Apache Foundation kind of way. We talked before about organizations that'd make sense for a parent; did that go anywhere? Is there anyone working on OA who would drive it somewhere?

**Comment by migurski** [comment]
I don’t believe it has. Waldo seems like an obvious person to ask about this, but I still think there’s a benefit to incorporating before we talk to a possible parent org; it gives us an opportunity to draw a circle around OA and figure out what’s inside.

**Comment by riordan** [comment]
Seconding the idea of LLC-ing then investigating a nonprofit shelter. Tax deductibility is nice to have. Liability umbrella's essential.

I'd be happy to help investigate incorporation.

**Comment by iandees** [comment]
To be clear, I wasn't suggesting a non-profit shelter because I was seeking non-profitness. It seems like a non-profit shelter would give us liability shield, entity for donation/owning, and a structure while also giving us non-profitness. It wasn't clear to me what the middle step was buying us. It sounds like it's buying us portability.

**Comment by dianashk** [comment]
I think it buys **legitimacy**. It's hard to accept donations or support when it's unclear where the funds are going, how they will be distributed, and even whether or not this project will be around tomorrow. It's hard to hire staff, part-time/interns/full-time, when there is no organization name to put on their paycheck. Having structure around this collection of humans/technology/data would help define what it actually is and how others can interact with it.

Since I'm not a lawyer, I don't know if there are better mechanisms for implementing the structure needed to gain legitimacy and accept support. I think even if you were going to be absorbed by an existing non-profit org, they would need to know what they are absorbing. They would need transparency into the who, what, where, etc. and most likely need you to be some sort of a business already.

**Comment by NelsonMinar** [comment]
I agree that it adds legitimacy. But all these steps get legitimacy by OA stepping up and becoming a legitimate organization with leadership and plans and decision making. Right now we're a loose casual affiliation based on goodwill. That's suited us pretty well but is a barrier to growth / getting more serious.

**Comment by migurski** [comment]
Yes, agreed. If we were to go the LLC route, I believe that VA and CA are the two states where we might reasonably incorporate. I am personally willing to do the footwork in CA.

**Comment by iandees** [comment]
As of July 2016 we have an LLC: https://sccefile.scc.virginia.gov/Business/S629413

**Comment by migurski** [comment]
Huzzah! We should add information about this to the OA site.
